ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -771,3 +762,24 @@ func convertTaskDefParams(ecsParams *ECSParams) (params TaskDefParams, e error)
 
 	return params, nil
 }
+
+func ConvertToLogConfiguration(inputCfg *config.ServiceConfig) (*ecs.LogConfiguration, error) {
```

Comment:
since these funcs are taking in a libcompose ServiceConfig as a param, is the idea that these will be eventually pulled into v1/2 parsing portion of the 'Project' package, or did you want to keep them here?